SlicerLite/SlicerLiteLib/UIUtils.py

```diff
@@ -0,0 +1,40 @@
+import qt, ctk
+
+class UIUtils:
```

Comment:
If everything is static this could be just a python file with all functions flat.